### PR TITLE
Fix swagger annotation on delete miner endpoint

### PR DESCRIPTION
**Pull request by gmelodie**

### CODE REVIEW

File: docs/docs.go [hunk 1]
```diff
@@ -1368,7 +1368,7 @@ const docTemplate = `{
                         "type": "string",
                         "description": "Filter by miner",
                         "name": "miner",
-                        "in": "query"
+                        "in": "path"
```
Comment (alvin-reyes) [inline docs/docs.go:1371]: this is automatic. all the swagger files (docs/*) are auto-generated after the swagger workflow job step.

**Review by alvin-reyes — CHANGES_REQUESTED** [review]
the docs.go will be regenerated after it runs the swag.sh file.

File: docs/docs.go (hunk 1 above)
Comment (gmelodie) [reply docs/docs.go:1371]: I didn't change that file at all. Maybe it was a pre-commit hook or something?

Comment (gmelodie) [reply docs/docs.go:1371]: It's a workflow on github: https://github.com/application-research/estuary/blob/a7420a2c22262bd6dcc1edb203eba6bb0f763afe/.github/workflows/swagger.yml#L18-L22

**Comment by gmelodie** [comment]
@alvin-reyes I don't know what you mean. The commit that changed `docs.go` was not mine, it was created by the workflow we have on the repo

**Review by alvin-reyes — APPROVED** [review]
LGTM

**Pull request merged**